### PR TITLE
Add repo SECURITY.md

### PR DESCRIPTION
**Pull request by joelsmith**
See https://github.com/kubernetes/security/issues/105

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+## Reporting a Vulnerability
+
+Instructions for reporting a vulnerability can be found on the
+[Kubernetes Security and Disclosure Information] page.
+
+## Supported Versions
+
+Information about supported Kubernetes versions can be found on the
+[Kubernetes version and version skew support policy] page on the Kubernetes website.
```
Comment (johngmyers) [inline SECURITY.md:17]: The Kubernetes version and version skew support policy is less relevant for this repo. More relevant would be [Releases and versioning](https://kops.sigs.k8s.io/welcome/releases/), as referenced in our README.

Comment (joelsmith) [reply SECURITY.md:17]: @johngmyers  Sorry about that. I'm just pushing the same generic file to all our repos, but it makes sense for kops to reference its own policy as appropriate. How would you like to proceed? Do you think we should merge this and then kops maintainers can edit as you see fit in follow-on PRs? Or do you want to propose an alternate PR now and I'll close this? Or do you want to give me edits and I'll make them in this PR?

Comment (johngmyers) [inline SECURITY.md:17]: It looks like Netlify flaked, so this PR will need another push to get a retry.

```suggestion
Information about supported Kops versions and the Kubernetes versions they support can be found on the 
[Releases and versioning](https://kops.sigs.k8s.io/welcome/releases/) page. Information about supported Kubernetes versions can be found on the 
[Kubernetes version and version skew support policy] page on the Kubernetes website.
```

**Comment by johngmyers** [comment]
/lgtm

Argh, Netlify is being stubborn.

**Pull request merged**